---
annotations:
- id: PW:0000004
  parent: regulatory pathway
  type: Pathway Ontology
  value: regulatory pathway
authors:
- Fehrhart
- Susan
- Mkutmon
- MaintBot
- Egonw
citedin:
- link: PMC9300967
  title: Shared Genetic Risk Factors Between Cancer and Cardiovascular Diseases (2022)
- link: PMC5123333
  title: Rett syndrome â€“ biological pathways leading from MECP2 to disorder phenotypes
    (2016)
communities:
- ONTOX
description: DNA, namely cytosine, methylation is the key event in epigentics. The
  degree of methylation influences gene expression and methylation disorders are known
  to be major causes of some diseases like Rett syndrome and some cancer types. Epigenetic
  effects also contribute to the development of Alzheimers' disease, developmental
  retardation by alcohol, Huntington's disease and ischemia-reperfusion injury. Methylation
  is not a static event but a highly dynamic and therefore highly regulated procedure.
  Cytosine is methylated by DNA methyltransferases (DNMTs) forming 5-methylcytosine
  (5mC). In a second step, 5mC is transformed to 5-hydroxymethylcytosine (5hmC) by
  ten-eleven-translocation enzymes (TET1-3). These enzymes are sensitive to regulation
  by a variety of metabolites (ethanol, a-ketoglutarate, 2-hydroxyglutarate), miRNA
  and MeCP2 (targeting directly TET1). Proteins binding methylated DNA like MeCP2
  or Mbd3 (as part of the NURD complex) also inhibit the conversion by blocking the
  target. MeCP2 also binds on 5hmC and block the transition to 5-formylcytosine (5fC)
  which is also catalyzed by the TET enzymes. The conversion back to cytosine is done
  by tymine DNA glycosylase (TGD) and base excision repair mechanism either directly
  or over another TET catalyzed step forming 5-carboxylcytosine (5caC).
last-edited: 2019-09-17
ndex: eba164d5-8b66-11eb-9e72-0ac135e8bacf
organisms:
- Homo sapiens
redirect_from:
- /index.php/Pathway:WP3585
- /instance/WP3585
- /instance/WP3585_r123390
revision: r123390
schema-jsonld:
- '@context': https://schema.org/
  '@id': https://wikipathways.github.io/pathways/WP3585.html
  '@type': Dataset
  creator:
    '@type': Organization
    name: WikiPathways
  description: DNA, namely cytosine, methylation is the key event in epigentics. The
    degree of methylation influences gene expression and methylation disorders are
    known to be major causes of some diseases like Rett syndrome and some cancer types.
    Epigenetic effects also contribute to the development of Alzheimers' disease,
    developmental retardation by alcohol, Huntington's disease and ischemia-reperfusion
    injury. Methylation is not a static event but a highly dynamic and therefore highly
    regulated procedure. Cytosine is methylated by DNA methyltransferases (DNMTs)
    forming 5-methylcytosine (5mC). In a second step, 5mC is transformed to 5-hydroxymethylcytosine
    (5hmC) by ten-eleven-translocation enzymes (TET1-3). These enzymes are sensitive
    to regulation by a variety of metabolites (ethanol, a-ketoglutarate, 2-hydroxyglutarate),
    miRNA and MeCP2 (targeting directly TET1). Proteins binding methylated DNA like
    MeCP2 or Mbd3 (as part of the NURD complex) also inhibit the conversion by blocking
    the target. MeCP2 also binds on 5hmC and block the transition to 5-formylcytosine
    (5fC) which is also catalyzed by the TET enzymes. The conversion back to cytosine
    is done by tymine DNA glycosylase (TGD) and base excision repair mechanism either
    directly or over another TET catalyzed step forming 5-carboxylcytosine (5caC).
  keywords:
  - 2-hydroxyglutarate
  - 5-carboxylcytosine
  - 5-formylcytosine
  - 5-hydroxymethylcytosine
  - 5-methylcytosine
  - DNMT1
  - IDH1
  - IDH2
  - MBD3
  - MECP2
  - TDG
  - TET1
  - TET2
  - TET3
  - a-ketoglutarate
  - cytosine
  - ethanol
  - isocitrate
  license: CC0
  name: Cytosine methylation
seo: CreativeWork
title: Cytosine methylation
wpid: WP3585
---